model/relabel/relabel.go

```diff
@@ -100,6 +100,8 @@ type Config struct {
 	Replacement string `yaml:"replacement,omitempty" json:"replacement,omitempty"`
 	// Action is the action to be performed for the relabeling.
 	Action Action `yaml:"action,omitempty" json:"action,omitempty"`
+	// NameValidationScheme to use when validating labels.
+	NameValidationScheme model.ValidationScheme `yaml:"-" json:"-"`
```

Comment:
Is the idea that this always inherits from the overall validation scheme, so it shouldn't be set via yaml/json?